### PR TITLE
#35352 Add Arabic language to installation and preferences

### LINKED ISSUE #35352: Add Arabic language to installation and preferences
We need to add Arabic to the language list in Preferences and Installation.
Before doing this, we need to do the following:

- [ ] verify the new [Arabic localization](https://github.com/dbeaver/dbeaver/issues/35128
- [ ] test rtl mode before  https://github.com/dbeaver/dbeaver/issues/11180

### PR DESCRIPTION
**Pull request by ivgag**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.registry/OSGI-INF/l10n/bundle.properties [hunk 1]
```diff
@@ -16,18 +16,19 @@ extension-point.org.jkiss.dbeaver.task.name = Tasks and task types
...
+language.fr.label=Fran\u00e7ais
+language.de.label=Deutsch
+language.es.label=Espa\u00f1ol
+language.pt_BR.label=Portugu\u00eas (BR)
+language.it.label=Italiano
+language.ja.label=\u65e5\u672c\u8a9e
+language.uk.label=\u0423\u043a\u0440\u0430\u0457\u043d\u0441\u044c\u043a\u0430
+language.ko.label=\ud55c\uad6d\uc5b4
+language.ro.label=Rom\u00e2n\u0103
+language.ar.label=\u0627\u0644\u0639\u0631\u0628\u064a\u0629
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.registry/OSGI-INF/l10n/bundle.properties]: Maybe we should hardcode them in the `plugin.xml` and add a comment stating these shouldn't be translated?

Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.registry/OSGI-INF/l10n/bundle.properties]: Remove these

File: plugins/org.jkiss.dbeaver.registry/plugin.xml [hunk 2]
```diff
@@ -22,19 +22,20 @@
...
-        <language code="ja" label="%language.ja.label"/>
-        <language code="ko" label="%language.ko.label"/>
-        <language code="pt_BR" label="%language.pt_BR.label"/>
-        <language code="ro" label="%language.ro.label"/>
-        <language code="ru" label="%language.ru.label"/>
-        <language code="zh" label="%language.zh.label"/>
-        <language code="es" label="%language.es.label"/>
-        <language code="tw" label="%language.tw.label"/>
-        <language code="uk" label="%language.uk.label"/>
+        <language code="en" label="English"/>
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.registry/plugin.xml:26]: Add a comment saying something like

```suggestion
        <!-- Language names should be written in their respective languages. Do not localize them -->
        <language code="en" label="English"/>
```

**Review by ShadelessFox — APPROVED** [review]

**Review by E1izabeth — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**